exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/ReplaceColumnTests.kt

```diff
@@ -14,7 +15,7 @@ class ReplaceColumnTests : DatabaseTestsBase() {
         withTables(IDTable) {
             expectException<DuplicateColumnException> {
                 // Duplicate the id column by replacing the IDTable.code by a column with the name "id"
-                val id = Column<Int>(IDTable, IDTable.id.name, IDTable.id.columnType)
+                val id = Column<Int>(IDTable, IDTable.id.name, (IDTable.id.columnType as EntityIDColumnType<Int>).idColumn.columnType)
```

Comment:
Could you tell me if this can be simplified?

Reply:
It doesn't seem like it can be simplified. Casting it to `EntityIDColumnType` is necessary to be able to get the underlying wrapped column storing the identity value.